src/System.Data.SqlClient/src/Microsoft/SqlServer/Server/SmiMetaData.cs

```diff
@@ -139,7 +139,7 @@ internal static SmiMetaData DefaultNVarChar
         }
 
 
-        // SMI V100 (aka V3) constructor.  Superceded in V200.
```

Comment:
For the most part I've left variants alone, but supercede is relatively rare and sometimes considered incorrect. While I don't consider it so, I decided to change it:

![Citations for supercede and supersede](https://books.google.com/ngrams/chart?content=supercede%2C+supersede&year_start=1800&year_end=2000&corpus=15&smoothing=3&share=&direct_url=t1%3B%2Csupercede%3B%2Cc0%3B.t1%3B%2Csupersede%3B%2Cc0)